client/src/controllers/PreviewController.ts

```diff
@@ -787,8 +787,8 @@ export class PreviewController extends Controller<HTMLElement> {
     }
 
     this.contentChecksPromise = (async () => {
-      await runContentChecks();
       await runAccessibilityChecks(() => this.newTabTarget.click());
+      await runContentChecks();
```

Comment:
Run content checks after accessibility checks so that `axe.configure()` (which happens in `getA11yReport`) has been run.

I suppose the ideal refactoring would make it in a way that ensures `axe.configure()` has been called once (and only once) regardless whether you run accessibility checks or content checks first.

Maybe something like having a dedicated function to run `axe.configure()` with the available config, that is called during initialisation. Then, any checks can just call `runAccessibilityChecks`/`runContentChecks` as necessary. We'll need to ensure that function is called once in PreviewController's init and Userbar's init.

For now I just opted for a simpler refactoring to ease the review.